javascript/javascript3/week3/promise/new.js

```diff
@@ -0,0 +1,23 @@
+let repolist = document.getElementById("repolist");
+    function render(userPromise) {
+        const itemsArray = userPromise.map(promise => promise.items);
+        console.log(itemsArray);
+        itemsArray.forEach(items => {
+            let fullName = document.createElement("li");
+            fullName.innerHTML = "<br>" + "Full name of repository : " + items[0].full_name + "<br>" + "owner of the repository : " +
+                items[0].owner.login + "<br>";
+            repolist.appendChild(fullName)
+            let a = document.createElement("a");
+            a.href = items[0].url;
+            a.innerHTML = "click here to go to that repository"
+            fullName.appendChild(a);
```

Comment:
Nice way you found by using <br> :)  I never saw this type of styling :)
instead we can also do
let owner = document.createElement("p");
owner.innerHTML =  "owner of the repository : " +items[0].owner.login;
repolist.appendChild(owner)